travis-scripts/run-sanity-tests

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+PROJECT_PATH=$(cd "$( dirname "$0" )/../" && pwd)
+
+/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_10.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :10 -ac -screen 0 1600x1200x16
```

Comment:
Is the resolution OK for our tests ?

Reply:
Yes, our tests passes with this size, and we define our browser size inside helper.
https://travis-ci.org/PrestaShop/PrestaShop/builds/585502141